.evergreen/config.yml

```diff
@@ -251,7 +251,7 @@ functions:
         working_dir: src/go.mongodb.org/mongo-driver
         script: |
           ${PREPARE_SHELL}
-          ${BUILD_ENV|} make ${targets} BUILD_TAGS="-tags cse,gssapi"
+          ${BUILD_ENV|} make ${targets} BUILD_TAGS="-tags \"cse gssapi\""
```

Comment:
Because these tags will now be used on 1.9 and 1.10 in addition to 1.13, we have to go back to the legacy way of specifying multiple tags delimited by spaces rather than commas. The escaped quotes are necessary because otherwise the `make` command interprets the second tag as a `make` target.